x/emissions/module/rewards_v2.go

```diff
@@ -0,0 +1,41 @@
+package module
+
+import (
+	"fmt"
+	"math"
+)
+
+// GetWorkerScore calculates the worker score based on the losses and lossesCut.
+func GetWorkerScore(losses, lossesCut float64) float64 {
+	deltaLogLoss := math.Log10(lossesCut) - math.Log10(losses)
+	return deltaLogLoss
+}
+
+// GetStakeWeightedLoss calculates the stake-weighted average loss.
+func GetStakeWeightedLoss(reputersStakes, reputersReportedLosses []float64) (float64, error) {
```

Comment:
what formula does this correspond to in the paper? I think it's a bad idea to put the formula number because those might change as the whitepaper gets edited, but maybe copy the latex into the comment is my feeling